packages/compass-sidebar/src/components/sidebar.tsx

```diff
@@ -134,57 +133,69 @@ export function Sidebar({
     [globalAppRegistryEmit]
   );
 
+  // TODO: how do we get notified about the toggle-sidebar app registry event and then get it here to pass to the resizable sidebar? or in the sidebar somehow?
```

Comment:
☝️ 

I suppose I can manage isExpanded from the outside and pass it in to the component? But that also feels weird.